app/assets/javascripts/admin/addon/lib/template-form-fields.js

```diff
@@ -0,0 +1,69 @@
+export const templateFormFields = [
+  {
+    type: "checkbox",
+    structure: `- type: checkbox
+  choices:
+    - "Option 1"
+    - "Option 2"
+    - "Option 3"
+  attributes:
+    label: "Enter question here"
```

Comment:
It can be for later, but these strings here will need translations.

Reply:
Good catch, added a TODO comment so I don't forget.